### PR TITLE
Add change log for Repeat Records Couch to SQL migration

### PR DESCRIPTION
**Pull request by millerdev**
https://dimagi.atlassian.net/browse/SAAS-14299

##### Environments Affected
None. This is for self-hosted environments. The migration has been completed on all of our internally managed environments.

##### Announce New Release
- [x] After merging, I will follow [these instructions](https://confluence.dimagi.com/display/saas/Announcing+changes+affecting+third+parties#Announcingchangesaffectingthirdparties-announcing) to announce a new commcare-cloud release.

### CODE REVIEW

File: changelog/0077-sql-repeatrecord-pr1.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  ```
+
+  The output should include something like the following:
+
+  ```
+  Records to migrate: 125011
+  ```
+
+  A future deploy will be blocked, requiring manual migration steps, if the
+  number of records to migrate is more than 1000. These steps can be skipped
```
Comment (gherceg) [inline changelog/0077-sql-repeatrecord-pr1.yml:34]: nit: Make the reader aware of whether this applies to them by saying how many records the threshold is first.
```suggestion
  If the number of records to migrate is more than 1000, a future deploy will be blocked, requiring manual migration steps. These steps can be skipped
```

File: changelog/0077-sql-repeatrecord-pr1.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  Next, estimate the amount of free space that will be required for new data in
+  PostgreSQL (main or repeaters database, depending on configuration):
+
+  ```sh
+  cchq <env> django-manage shell -c '
+  from corehq.motech.repeaters.models import RepeatRecord;
+  size = RepeatRecord.get_db().info()["sizes"]["active"];
+  mult = int((int.bit_length(size) - 1) / 10);
+  unit = " kMGT"[mult].strip();
```
Comment (gherceg) [inline changelog/0077-sql-repeatrecord-pr1.yml:51]: what is " kMGT" out of curiosity?

File: changelog/0077-sql-repeatrecord-pr1.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  # Run the migration command
+  ./manage.py populate_repeatrecords --log-path ~/couch2sql
+  # Re-run this command until it completes successfully (exit code 0).
+
+  # After successful completion, run again with --fixup-diffs=... if directed
+  # to do so by command output.
+  ```
+
+  DO NOT delete repeat records from Couch DB yet, they are still needed! A
+  future change log entry will instruct when it is safe to to that.
```
Comment (gherceg) [inline changelog/0077-sql-repeatrecord-pr1.yml:94]:
```suggestion
  future change log entry will instruct when it is safe to do that.
```

File: changelog/0077-sql-repeatrecord-pr1.yml (hunk 2 above)
Comment (millerdev) [reply changelog/0077-sql-repeatrecord-pr1.yml:51]: Single-letter versions of kilo, Mega, Giga, Tera prefixes. I guess 'k' could have been capitalized.

File: changelog/0077-sql-repeatrecord-pr1.yml [hunk 4]
```diff
@@ -0,0 +1,104 @@
+title: 'SQL Repeat Record Migration'
+key: sql-repeatrecord-pr1
+date: 2024-02-13
+optional_per_env: yes
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version: TODO
```
Comment (gherceg) [inline changelog/0077-sql-repeatrecord-pr1.yml:6]: You will need to fill this out before merging right?

**Review by gherceg — APPROVED** [review]

**Comment by millerdev** [comment]
[Release announcement](https://forum.dimagi.com/t/commcare-cloud-release-notes-2024-03-19/10653)

**Pull request merged**